cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PatternTest.scala

```diff
@@ -207,6 +213,7 @@ class PatternTest extends DocumentingTest {
         """Note that this form of pattern can only be used to describe existing data (ie. when using a pattern with `MATCH` or as an expression).
           |It will not work with `CREATE` or `MERGE`, since it's not possible to create a relationship with multiple types."""
       )
+      p("""For more information on use relationship type expressions see <<query-syntax-type, here>>""")
```

Comment:
Not sure if this is correct, but I was aiming to make reading easier:

```suggestion
      p("""For more information on how to use Relationship Type Expressions see <<query-syntax-type, here>>""")
```

Reply:
^